src/client/camera.cpp

```diff
@@ -45,6 +45,8 @@ with this program; if not, write to the Free Software Foundation, Inc.,
 #define WIELDMESH_AMPLITUDE_X 7.0f
 #define WIELDMESH_AMPLITUDE_Y 10.0f
 
+#define HANDS (int i = 0, s = +1; i <= 1; i++, s *= -1) // i is index, s is sign
```

Comment:
`s` is only used in one location. Please keep this macro as short as possible. It already looks like bad practice. The cleanest solution might be to offload the wieldhand code to a separate class + methods (not a requirement from my side as of now).